README.adoc

```diff
@@ -0,0 +1,162 @@
+|https://registry.terraform.io/providers/hashicorp/oci/latest/docs/data-sources/core_services[oci_core_services.all] |data source
+|https://registry.terraform.io/providers/hashicorp/oci/latest/docs/data-sources/core_services[oci_core_services.storage] |data source
+|https://registry.terraform.io/providers/hashicorp/oci/latest/docs/data-sources/identity_compartments[oci_identity_compartments.network] |data source
+|===
+
+== Inputs
+
+[cols="a,a,a,a,a",options="header,autowidth"]
+|===
+|Name |Description |Type |Default |Required
+|[[input_input]] <<input_input,input>>
+|Resources identifier from resident module
+|
+
+[source]
+----
+object({
+      internet = string,
+      nat      = string,
+      ipv6     = bool,
+      osn      = string
+    })
+----
+
+|n/a
+|yes
+
+|[[input_assets]] <<input_assets,assets>>
+|Retrieve asset identifier
+|
+
+[source]
+----
+object({
+    resident = any
+  })
+----
+
+|n/a
+|yes
+
+|[[input_tenancy]] <<input_tenancy,tenancy>>
+|Tenancy Configuration
+|
+
+[source]
+----
+object({
+    id      = string,
+    class   = number,
+    buckets = string,
+    region  = map(string)
+  })
+----
+
+|n/a
+|yes
+
+|[[input_resident]] <<input_resident,resident>>
+|Service Configuration
+|
+
+[source]
+----
+object({
+    owner          = string,
+    name           = string,
+    label          = string,
+    stage          = number,
+    region         = map(string)
+    compartments   = map(number),
+    repository     = string,
+    groups         = map(string),
+    policies       = map(any),
+    notifications  = map(any),
+    tag_namespaces = map(number),
+    tags           = any
+  })
+----
+
+|n/a
+|yes
+
+|[[input_network]] <<input_network,network>>
+|Network Configuration
+|
+
+[source]
+----
+object({
+    name         = string,
+    region       = string,
+    display_name = string,
+    dns_label    = string,
+    compartment  = string,
+    stage        = number,
+    cidr         = string,
+    gateways     = any,
+    route_tables = map(any),
+    subnets      = map(any),
+    security_lists = any
+  })
+----
+
+|n/a
+|yes
+
+|===
+
+== Outputs
+
+[cols="a,a",options="header,autowidth"]
+|===
+|Name |Description
+|[[output_compartment_id]] <<output_compartment_id,compartment_id>> |OCID for the network compartment
+|[[output_vcn_id]] <<output_vcn_id,vcn_id>> |Identifier for the Virtual Cloud Network (VCN)
+|[[output_gateways]] <<output_gateways,gateways>> |A list of gateways for the Virtual Cloud Network (VCN)
+|[[output_subnets]] <<output_subnets,subnets>> |A list of subnets for the Virtual Cloud Network (VCN)
+|[[output_route_tables]] <<output_route_tables,route_tables>> |A list of route_tables for the Virtual Cloud Network (VCN)
+|[[output_security_lists]] <<output_security_lists,security_lists>> |All security lists defined for the Virtual Cloud Network (VCN)
+|===
```

Comment:
Please add the license/disclaimer text here (copy-and-paste from the MD version of the README).

Reply:
Done